ArchiSteamFarm/Bot.cs

```diff
@@ -2836,6 +2835,194 @@ public sealed class Bot : IAsyncDisposable {
 			}
 		}
 
+		private void OnInventoryChanged() {
+			Utilities.InBackground(CardsFarmer.OnNewItemsNotification);
+
+			if (BotConfig.BotBehaviour.HasFlag(BotConfig.EBotBehaviour.DismissInventoryNotifications)) {
+				Utilities.InBackground(ArchiWebHandler.MarkInventory);
+			}
+
+			if (BotConfig.SendSetsOnCompleted) {
+				Utilities.InBackground(SendCompletedSets);
```

Comment:
Both other functions that you call have logic for being called multiple times, your function does not and it can't stay like that - you can have 100 inventory changed events across 3 seconds, assuming that your function takes more than 3s to complete, you're not expected to do that logic more than 2 times total.